operations/CSM_product_management/Post_Install_Customizations.md

```diff
@@ -374,90 +286,59 @@ Refer to the note at the end of this section for more details.
-
-      ```json
-      {
-      "limits": {
-         "cpu": "8",
-         "memory": "8Gi"
-      },
-      "requests": {
-         "cpu": "4",
-         "memory": "4Gi"
-      }
-      }
-      ```
-
-1. (`ncn-mw#`) **This step is critical.** Store the modified `customizations.yaml` file in the `site-init` repository in the customer-managed location.
+Verify that the pods restart and that the desired resources have been applied. Commands in this section  use the
+`CHART_NAME` variable which should have been set as part of the [Redeploying a Chart](Redeploying_a_Chart.md) procedure.
 
-   If this is not done, these changes will not persist in future installs or upgrades.
+1. (`ncn-mw#`) Set the `ns` variable to the name of the appropriate Kubernetes namespace from the earlier table.
 
-   ```bash
-   kubectl delete secret -n loftsman site-init
-   kubectl create secret -n loftsman generic site-init --from-file=customizations.yaml
-   ```
-
-**IMPORTANT:** If `cray-sls-postgres`, `cray-smd-postgres`, or `gitea-vcs-postgres` resources need to be adjusted,
-the same procedure as above can be used with the following changes:
-
-> **NOTE:** When copying bash commands from the above section, please use a YAML file name that matches the corresponding
-> cached manifest ConfigMap to avoid accidentally using a wrong YAML file.
-
-* `cray-sls-postgres`
+    ```bash
+    nc=<put kubernetes namespace here>
+    ```
 
-  * Get the current cached manifest ConfigMap from: `loftsman-core-services`
-  * Resource path: `spec.kubernetes.services.cray-hms-sls.cray-postgresql.sqlCluster.resources`
+1. (`ncn-mw#`) Watch the pod restart.
 
-* `cray-smd-postgres`
+    ```bash
+    watch "kubectl get pods -n ${ns} -l application=spilo,cluster-name=${CHART_NAME}"
+    ```
 
-  * Get the current cached manifest ConfigMap from: `loftsman-core-services`
-  * Resource path: `spec.kubernetes.services.cray-hms-smd.cray-postgresql.sqlCluster.resources`
+1. Verify that the desired resources have been applied.
 
-* `gitea-vcs-postgres`
+    ```bash
+    kubectl get pod ${CHART_NAME}-0 -n "${ns}" -o json | jq -r '.spec.containers[] | select(.name == "postgres").resources'
+    ```
 
-  * Get the current cached manifest ConfigMap from: `loftsman-sysmgmt`
-  * Resource path: `spec.kubernetes.services.gitea.cray-postgresql.sqlCluster.resources`
+    Example output:
+
+    ```json
+    {
+    "limits": {
+       "cpu": "8",
+       "memory": "8Gi"
+    },
+    "requests": {
+       "cpu": "4",
+       "memory": "4Gi"
+    }
+    }
+    ```
 
 ### Scale `cray-bss` service
 
 Scale the replica count associated with the `cray-bss` service in the `services` namespace.
 This example is based on what was needed for a system with 4000 compute nodes.
 Trial and error may be needed to determine what is best for a given system at scale.
 
-1. (`ncn-mw#`) Get the current cached customizations.
+Follow the [Redeploying a Chart](Redeploying_a_Chart.md) procedure to redeploy the
```

Comment:
This also feels like it's too early vs the steps provided.

Reply:
This is why it tells you to keep both pages open in the browser. You are following that procedure, but you need information from this page in order to do it. The alternatives to having it be in this format are:

1. Don't document the procedure in a central place, and instead repeat that procedure in all places where we want to charts to be redeployed. The downside of this approach is that we do this in a non-trivial number of places in the docs, and historically we have not done a good job of making sure that changes to a procedure get propagated to every place where that procedure appears in the docs. 

2. As you noted in another comment, automate this stuff. I'll respond to that comment in its own thread.

Reply:
I will try to add some language and markup that make it much clearer that you need to be following the general procedure and only use the other sections when you get to those parts.